cmd/bosun/sched/sched.go

```diff
@@ -704,6 +725,111 @@ func (a ActionType) MarshalJSON() ([]byte, error) {
 	return json.Marshal(a.String())
 }
 
+type Incident struct {
+	Id       uint32
+	Start    time.Time
+	End      time.Time
+	AlertKey expr.AlertKey
+}
+
+func (s *Schedule) createIncident(ak expr.AlertKey, start time.Time) *Incident {
+	s.incidentLock.Lock()
+	defer s.incidentLock.Unlock()
+	if s.Incidents == nil {
```

Comment:
Look at https://github.com/bosun-monitor/bosun/blob/502e6253e846bd97b5ea1cb1998c4c7fa95fa776/cmd/bosun/sched/sched.go#L366

We already initialize a whole pile of maps there.